content/java/concepts/math-methods/terms/sqrt/sqrt.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.sqrt()'
+Description: 'Returns the square root of a double value passed to it as an argument.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+  - 'Sqrt'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.sqrt()`** method returns the square root of a double value passed to it as an argument.
+
+## Syntax
+
+```pseudo
+Math.sqrt(double num)
+```
+
+The `.sqrt()` method takes one parameter.
```

Comment:
```suggestion
The return type of the `.sqrt()` method is a double. The data type of the `num` parameter is also a double. However, if a decimal is not provided, `num` will still be treated like a double-type.

Some edge cases for the `.sqrt()` method include the following:

 - If `num` is less than zero or it is not a numeric type (`NaN`), then `NaN` is returned.
 - If `num` is positive infinity (`Double.POSITIVE_INFINITY`), then `Infinity` is returned. 
```

The "Syntax" section is a good place to list out, if any, the edge cases of parameters used in a method/function.